### PR TITLE
Adding Chicago,  IL file

### PR DESCRIPTION
**Pull request by zzbarrao**
Adding first time to the file zzbarrao-chicago.  I will be adding more information later.

@kairudloff

### CODE REVIEW

File: zzbarrao-chicago.rd [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Chicago Recommentations
+
+### Restraunts
+Bob Chinns
```
Comment (zzbarrao) [inline zzbarrao-chicago.rd:4]: How about Alexanders?

**Review by zzbarrao — COMMENTED** [review]
Reviwed

**Review by discoversmo — APPROVED** [review]
doubly approved

**Review by githubteacher — APPROVED** [review]
:shipit:

**Review by kairudloff — APPROVED** [review]
Nice!

**Review by kairudloff — COMMENTED** [review]
Please add the Ram.

**Pull request merged**